etc/testing/circle/kube_debug.sh

```diff
@@ -42,6 +43,29 @@ cmds=(
   'kubectl logs --namespace=test-cluster-6 --tail=1500 -l suite=pachyderm --all-containers=true --prefix=true'
   # if pachd restarted
   'kubectl logs --tail=1500 -l suite=pachyderm,app=pachd --previous --prefix=true'
+  'kubectl logs --namespace=test-cluster-1 --tail=1500 -l suite=pachyderm,app=pachd --previous --prefix=true'
```

Comment:
Any reason not to just grab suite=pachyderm?  Also, are there still 6 namespaces?  I feel like we might have dropped it down to 3 at some point.

Reply:
I'll switch to suite=pachyderm. I'm going to leave at 6 namespaces though. If I'm remembering right, I think there is a bug in minikubetestenv where the full-auto deployment and more manual InstallRelease count separately and we can end up with double the expected namespaces. I thought I wrote a ticket, but can't find it. I'll add one to the backlog if I can find it on a more thorough search.